BUILD.bazel

```diff
@@ -20,12 +20,36 @@ exports_files(["LICENSE"])
 # Protobuf Runtime Library
 ################################################################################
 
+cc_library(
+    name = "port_def",
+    hdrs = ["src/google/protobuf/port.h"],
+    includes = ["src"],
+    textual_hdrs = [
+        "src/google/protobuf/port_def.inc",
+        "src/google/protobuf/port_undef.inc",
+    ],
+    visibility = ["//:__subpackages__"],
+)
+
+cc_library(
+    name = "arena",
+    srcs = ["src/google/protobuf/arena.cc"],
+    hdrs = [
+        "src/google/protobuf/arena.h",
+        "src/google/protobuf/arena_impl.h",
```

Comment:
if _impl is a private header should it go in the srcs dir?

Reply:
Done.